source.go

```diff
@@ -195,7 +195,7 @@ func generateSourceFromImage(name string, st llb.State, cmd *Command, sOpts Sour
 	baseRunOpts := []llb.RunOption{CacheDirsToRunOpt(cmd.CacheDirs, "", "")}
 
 	for _, src := range cmd.Mounts {
-		srcSt, err := src.Spec.AsMount(name, sOpts, opts...)
```

Comment:
I think this may have been a bug from the refactor

Reply:
🤷 It happens, now we have a test case for it 😆